allennlp/modules/token_embedders/pretrained_transformer_embedder.py

```diff
@@ -39,8 +39,12 @@ class PretrainedTransformerEmbedder(TokenEmbedder):
         want to use the encoder.
     train_parameters: `bool`, optional (default = `True`)
         If this is `True`, the transformer weights get updated during training. If this is `False`, the
-        transformer weights are not updated during training and the dropout and batch normalization layers
-        are set to evaluation mode.
+        transformer weights are not updated during training.
```

Comment:
I used `e.g.,` here intentionally, since the next generation of pre-trained transformers might have other components that are eval- or training- mode dependent. I did make it more clear, though---is this what you were getting at? Also, changing `evaluated on development or train data.` to `evaluated on development or test data.`